types/index.test-d.ts

```diff
@@ -0,0 +1,85 @@
+import * as luxon from 'luxon';
+import { expectType } from 'tsd';
+import * as cron from '..';
+
+const CronJob = cron.CronJob;
+const CronTime = cron.CronTime;
+
+const DateTime = luxon.DateTime;
+
+const timeZone = 'America/Los_Angeles';
+
+// basic cron usage
+const job = new CronJob(
+  '* * * * * *',
+  () => {},
+  null,
+  true
+);
+
+// using factory function with parameters object
+expectType<cron.CronJob>(cron.job({
+  cronTime: '00 30 11 * * 1-5',
+  onTick: () => {},
+  start: false,
+  timeZone: 'America/Los_Angeles'
+}));
+
+// with timezone
+new CronJob(
```

Comment:
with these instantiations that aren't assigned to variables, is the idea that the test will fail if there's a type mismatch?

Reply:
Yes absolutely, that file is never actually ran, it's only statically analysed by the Typescript compiler